dice.js

```diff
@@ -0,0 +1,10 @@
+const rollDice = function(){
+
+let dicRoll1 = Math.floor(Math.random() * Math.floor(6))+1;
+let dicRoll2 = Math.floor(Math.random() * Math.floor(6))+1;
```

Comment:
You don't need to use `Math.floor()` with a **whole number**. The output of `Math.floor(6)` is `6`, so just type 6